pkg/externalip/registry/externalip/registry.go

```diff
@@ -0,0 +1,74 @@
+package externalip
+
+import (
+	"github.com/openshift/origin/pkg/externalip/api"
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/rest"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/watch"
+)
+
+// Registry is an interface for things that know how to store ExternalIP objects.
+type Registry interface {
```

Comment:
You don't need a registry anymore - we're trying to use the client interfaces if we need to use them internally.  You can remove this whole file.